PrebidMobile/API1.0/src/main/java/org/prebid/mobile/PrebidServerAdapter.java

```diff
@@ -469,24 +469,39 @@ private JSONObject getPostData() throws NoContextException {
                 if (ext != null && ext.length() > 0) {
                     postData.put("ext", ext);
                 }
+
+                JSONObject objectWithoutEmptyValues = Util.getObjectWithoutEmptyValues(postData);
```

Comment:
This part of code allows us to write an elegant code like:

``` Java
user.put("keywords", globalUserKeywordString);
ext.put("consent", TargetingParams.getGDPRConsentString());
```

instead of:
```Java
if (!TextUtils.isEmpty(finalKeywords)) {
    user.put("keywords", finalKeywords);
}
if (TargetingParams.isSubjectToGDPR() != null) {
    ext.put("consent", TargetingParams.getGDPRConsentString());
}
```

Step by step (during refactoring) I am going to remove redundant `if (!TextUtils.isEmpty())` checks